### PR TITLE
Smaller Features added/changed

### PR DESCRIPTION
**Pull request by TakoTheDev**

### CODE REVIEW

File: src/main/kotlin/skytils/skytilsmod/core/Config.kt [hunk 1]
```diff
@@ -774,6 +790,14 @@ object Config : Vigilant(File("./config/skytils/config.toml"), "Skytils", sortin
     )
     var changeAllSameColorTerminalSolver = false
 
+    @Property(
+        type = PropertyType.SELECTOR, name = "Change All to Same Color Solver Mode",
+        description = "Changes the display mode of the solver.",
+        category = "Dungeons", subcategory = "Terminal Solvers",
+        options = ["Normal", "Left/Right", "Monkey"]
```
Comment (My-Name-Is-Jeff) [inline src/main/kotlin/skytils/skytilsmod/core/Config.kt]: can you pick better names for this cause 99% of people won't know what Monkey is

File: src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/PartyFinderStats.kt [hunk 2]
```diff
@@ -155,21 +155,26 @@ object PartyFinderStats {
 
                             remove(null)
                         }
+
                         component.append(
                             UTextComponent("§dImportant Items: §7(Hover)\n\n").setHoverText(
-                                items.joinToString(
-                                    "§8, "
-                                )
+                                if(items.isEmpty()) {
```
Comment (My-Name-Is-Jeff) [inline src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/PartyFinderStats.kt]: code style

File: src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/PartyFinderStats.kt [hunk 3]
```diff
@@ -155,21 +155,26 @@ object PartyFinderStats {
...
+                                    )
+                                }
                             )
                         )
                     } ?: component.append("§cInventory API disabled!\n\n")
 
                     cataData.highestCompletion?.let { highestFloor ->
                         val completionObj = cataData.completions!!
                         component.append(UTextComponent("§aFloor Completions: §7(Hover)\n").setHoverText(buildString {
-                            for (i in 0 .. highestFloor) {
```
Comment (My-Name-Is-Jeff) [inline src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/PartyFinderStats.kt]: why?

File: src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/ScoreCalculation.kt [hunk 4]
```diff
@@ -220,18 +220,18 @@ object ScoreCalculation {
                 if (!hasSaid270 && score >= 270) {
                     hasSaid270 = true
                     if (Skytils.config.createTitleOn270Score) GuiManager.createTitle(
-                        Skytils.config.message270Score.ifBlank { "270 score" },
+                        "§c§l" + Skytils.config.messageTitle270Score,
```
Comment (My-Name-Is-Jeff) [inline src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/ScoreCalculation.kt]: you should set a default value

File: src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/ScoreCalculation.kt [hunk 5]
```diff
@@ -220,18 +220,18 @@ object ScoreCalculation {
...
                         20
                     )
-                    if (Skytils.config.sendMessageOn270Score) Skytils.sendMessageQueue.add("/pc Skytils-SC > ${Skytils.config.message270Score.ifBlank { "270 score" }}")
```
Comment (My-Name-Is-Jeff) [inline src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/ScoreCalculation.kt:226]: should not be removed

File: src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/ScoreCalculation.kt [hunk 6]
```diff
@@ -220,18 +220,18 @@ object ScoreCalculation {
...
+                    if (Skytils.config.sendMessageOn270Score) Skytils.sendMessageQueue.add("/pc ${Skytils.config.message270Score.ifBlank { "270 score" }}")
                 }
                 if (!hasSaid300 && score >= 300) {
                     hasSaid300 = true
                     if (Skytils.config.createTitleOn300Score) GuiManager.createTitle(
-                        Skytils.config.message300Score.ifBlank { "300 score" },
```
Comment (My-Name-Is-Jeff) [inline src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/ScoreCalculation.kt:231]: same as above

File: src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/ScoreCalculation.kt [hunk 7]
```diff
@@ -220,18 +220,18 @@ object ScoreCalculation {
...
+                        "§c§l" + Skytils.config.messageTitle300Score,
                         20
                     )
-                    if (Skytils.config.sendMessageOn300Score) Skytils.sendMessageQueue.add("/pc Skytils-SC > ${Skytils.config.message300Score.ifBlank { "300 score" }}")
```
Comment (My-Name-Is-Jeff) [inline src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/ScoreCalculation.kt:234]: same as above

File: src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/solvers/terminals/ChangeAllToSameColorSolver.kt [hunk 8]
```diff
@@ -50,15 +51,35 @@ object ChangeAllToSameColorSolver {
         }
         GlStateManager.translate(0f, 0f, 299f)
         for ((slot, clicks) in mapping) {
-            val betterOpt = if (clicks.first > -clicks.second) clicks.second else clicks.first
+            var betterOpt = if (clicks.first > -clicks.second) "${clicks.second}" else "${clicks.first}"
+            var color = CommonColors.WHITE
+            if(Skytils.config.changeToSameColorMode == 1){
+                val leftClick = when(Minecraft.getMinecraft().gameSettings.keyBindAttack.keyCode){
```
Comment (My-Name-Is-Jeff) [inline src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/solvers/terminals/ChangeAllToSameColorSolver.kt]: yeah i'm not really too sure about this one

**Review by My-Name-Is-Jeff — CHANGES_REQUESTED** [review]

File: src/main/kotlin/skytils/skytilsmod/core/Config.kt (hunk 1 above)
Comment (TakoTheDev) [reply src/main/kotlin/skytils/skytilsmod/core/Config.kt]: hoping chosen names are better

File: src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/solvers/terminals/ChangeAllToSameColorSolver.kt (hunk 8 above)
Comment (TakoTheDev) [reply src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/solvers/terminals/ChangeAllToSameColorSolver.kt:57]: Im gonna remove it for now

Comment (TakoTheDev) [reply src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/solvers/terminals/ChangeAllToSameColorSolver.kt]: Im gonna remove it ig

**Comment by TakoTheDev** [comment]
basically everything except "code style" has been fixed

File: src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/solvers/terminals/ChangeAllToSameColorSolver.kt (hunk 8 above)
Comment (TakoTheDev) [reply src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/solvers/terminals/ChangeAllToSameColorSolver.kt]: removed :cry:

File: src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/PartyFinderStats.kt (hunk 2 above)
Comment (TakoTheDev) [reply src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/PartyFinderStats.kt]: hopefully resolved?

**Review by My-Name-Is-Jeff — APPROVED** [review]

**Pull request merged**